metadata-jobs/mce-consumer-job/src/main/java/com/linkedin/metadata/kafka/MceConsumerApplication.java

```diff
@@ -8,13 +10,36 @@
 import org.springframework.boot.autoconfigure.elasticsearch.ElasticsearchRestClientAutoConfiguration;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.FilterType;
+import org.springframework.context.annotation.PropertySource;
 
 
 @SuppressWarnings("checkstyle:HideUtilityClassConstructor")
-@SpringBootApplication(exclude = {ElasticsearchRestClientAutoConfiguration.class, CassandraAutoConfiguration.class,
-    SolrHealthContributorAutoConfiguration.class})
-@ComponentScan(excludeFilters = {
-        @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = ScheduledAnalyticsFactory.class)})
+@SpringBootApplication(exclude = {
+        ElasticsearchRestClientAutoConfiguration.class,
+        CassandraAutoConfiguration.class,
+        SolrHealthContributorAutoConfiguration.class
+})
+@ComponentScan(basePackages = {
+        "com.linkedin.gms.factory.auth",
+        "com.linkedin.gms.factory.common",
+        "com.linkedin.gms.factory.config",
+        "com.linkedin.gms.factory.entity",
+        "com.linkedin.gms.factory.entityregistry",
+        "com.linkedin.gms.factory.kafka",
+        "com.linkedin.gms.factory.search",
+        "com.linkedin.gms.factory.secret",
+        "com.linkedin.gms.factory.timeseries",
+        "com.linkedin.restli.server",
+        "com.linkedin.metadata.restli",
+        "com.linkedin.metadata.kafka",
+        "com.linkedin.metadata.dao.producer"
+}, excludeFilters = {
+        @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = {
+                ScheduledAnalyticsFactory.class,
+                RestliEntityClientFactory.class
+        })
+})
+@PropertySource(value = "classpath:/application.yml", factory = YamlPropertySourceFactory.class)
```

Comment:
Thanks for adding this in

Reply:
Was this not working before? For some reason I vaguely remember thinking it was

Reply:
This definitely works with the current code, I think it was necessary as a part of the refactor?

Reply:
Yeah, there is a local application.properties it was not finding specific things in the application.yml required for restli so this now combines both.